brave-spancollector-scribe/src/main/java/com/github/kristofa/brave/scribe/SpanProcessingThread.java

```diff
@@ -83,8 +86,8 @@ public Integer call() {
                 final Span span = queue.poll(5, TimeUnit.SECONDS);
                 if (span == null) {
                     subsequentEmptyBatches++;
-
                 } else {
+                    notificationsHandler.onQueueSizeChanged(queue.size());
```

Comment:
if we ran a benchmark, this would pop up depending on the type of queue in use. This is why I like increment instead.